Sources/ClientRuntime/Logging/SDKLoggingSystem.swift

```diff
@@ -15,13 +15,13 @@ public class SDKLoggingSystem {
         factories[label] = logHandlerFactory
     }
 
-    public class func initialize() {
+    public class func initialize(defaultLogLevel: SDKLogLevel = .info) {
         LoggingSystem.bootstrap { label in
             if let factory = factories[label] {
                 return factory.construct(label: label)
             }
             var handler = StreamLogHandler.standardOutput(label: label)
-            handler.logLevel = .info
+            handler.logLevel = defaultLogLevel.toLoggerType()
             return handler
```

Comment:
What's the reason for this addition when there already exists an initializer that takes in a `SDKLogLevel` in lines 29 to 35?

Reply:
Because the other initializer ignores the manually set values. 
And the documentation of the bootstrap method says that bootstrap must be call only once otherwise it will likely crash

Reply:
Gotcha - just ran CIs, will merge now.